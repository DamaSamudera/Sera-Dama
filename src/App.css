* {
  animation: fade-in 0.5s ease-in-out;
  box-sizing: border-box;
  font-family: 'Montserrat', sans-serif;
  list-style: none;
  margin: 0;
  outline: none;
  padding: 0;
  scroll-behavior: smooth;
}

@keyframes fade-in {
  from {
    opacity: 0;
  }
  to {
    opacity: 1;
  }
}

a {
  color: inherit;
  text-decoration: none;
}

body {
  overflow-x: hidden;
  color: #333333;
}

h1 {
  font-size: 4.375rem;
  line-height: 5.125rem;
}

h2 {
  font-size: 3rem;
  line-height: 3.6875rem;
}

h3 {
  font-size: 2.125rem;
  line-height: 2.5625rem;
}

h4 {
  font-size: 1.5rem;
  font-weight: 600;
  line-height: 1.8125rem;
}

h5 {
  font-size: 1.25rem;
  font-weight: 600;
  line-height: 1.5rem;
}

h6 {
  font-size: 1rem;
  font-weight: 600;
  line-height: 1.25rem;
}

p {
  font-size: 0.875rem;
  line-height: 1.125rem;
}

#app {
  overflow-x: hidden;
  z-index: 1;
}

:root {
  --button: #2A844E;
  --dark-primary-color: #243a48;
  --error-color: #d0021b;
  --header-height: 0rem;
  --grey-color-1: #666666;
  --grey-color-2: #8c8c8c;
  --light-grey-color: #bdbdbd;
  --light-primary-color: #bed0a8;
  --light-text-color: #6d9686;
  --primary-color: #436e67;
  --secondary-color: #fab005;
  --success-color: #1ea54f;
  --text-color: #424242;
  --ultra-light-primary-color: #e6e4cf;
  --ultra-light-text-color: #f2f2f2;
  font-size: calc(100vw * 16 / 1440);
}

::-webkit-scrollbar {
  width: 0.625rem;
  height: 0.625rem;
}

::-webkit-scrollbar-thumb {
  border-radius: 1.5625rem;
  border: 0.0625rem solid #ffffff;
  background: rgba(36, 58, 72, 0.4);
  opacity: 0.1;
  position: absolute;
}

.secondary {
  background: none !important;
  border: 1px solid #333333 !important;
  border-radius: 4px !important;
  color: black !important;
}

.secondary:hover {
  background-color: var(--secondary-color) !important;
  border: 1px solid var(--secondary-color) !important;
}

@media only screen and (max-device-width: 667px) and (-webkit-min-device-pixel-ratio: 2) {
  :root {
    font-size: calc(100vw * 16 / 375);
    --header-height: 4rem;
  }
}
