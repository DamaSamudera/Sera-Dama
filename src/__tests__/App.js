import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import App from '../App';

jest.mock('react-hot-loader/root', () => ({ hot: c => c }));
jest.mock('../pages', () => ({ Login: 'login' }));

describe('src/App', () => {
  test('render', () => {
    const shallow = new ShallowRenderer();
    const store = {
      dispatch: jest.fn(),
      getState: jest.fn(),
      subscribe: jest.fn(),
    };
    const tree = shallow.render(<App history={{}} store={store} />);
    expect(tree).toMatchSnapshot();
  });
});
