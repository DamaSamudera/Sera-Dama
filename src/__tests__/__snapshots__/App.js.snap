// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/App render 1`] = `
<Provider
  store={
    Object {
      "dispatch": [MockFunction],
      "getState": [MockFunction],
      "subscribe": [MockFunction],
    }
  }
>
  <Router
    history={Object {}}
  >
    <AppContextProvider>
      <Switch>
        <Route
          exact={true}
          path="/login"
        />
        <Route
          exact={true}
          path="/dashboard"
        />
        <Route
          exact={true}
          path="/approval"
        />
        <Route
          exact={true}
          path="/company-list"
        />
        <Route
          exact={true}
          path="/pratinjau/:id"
        />
        <PageBase>
          <Route
            exact={true}
            path="/profile/:user"
          />
          <Route
            exact={true}
            path="/blog"
          />
          <Route
            exact={true}
            path="/search-blog/:query?"
          />
          <Route
            exact={true}
            path="/blog-content/:id"
          />
          <Route
            exact={true}
            path="/video-content/:id"
          />
          <Route
            exact={true}
            path="/"
          />
          <Route
            exact={true}
            path="/blog/video-list/:query?"
          />
          <Route
            exact={true}
            path="/calculator"
          />
          <Route
            exact={true}
            path="/create-blog"
          />
          <Route
            exact={true}
            path="/create-video"
          />
          <Route
            exact={true}
            path="/forum"
          />
          <Route
            exact={true}
            path="/forum-detail"
          />
          <Route
            exact={true}
            path="/profile-expert"
          />
          <Route
            exact={true}
            path="/expert"
          />
          <Route
            exact={true}
            path="/expert/:id"
          />
          <Route
            exact={true}
            path="/info-budidaya"
          />
          <Route
            exact={true}
            path="/info-budidaya-detail"
          />
          <Route
            exact={true}
            path="/kebijakan-privasi"
          />
        </PageBase>
        <Route />
      </Switch>
    </AppContextProvider>
  </Router>
</Provider>
`;
